### PR TITLE
Demo feedback items

### PR DESCRIPTION
**Pull request by johnwcallahan**
## Description

- AddNewMenu: "Firewalls" -> "Firewall".
- LinodeEntityDetail: "Launch Console" -> "LISH Console".
- Fix "Bucket" menu option in the AddNewMenu.

I'm not thrilled about about the repeated code for the route-matching to open drawers, but I figure it's fine for now since the Create Drawers may be replaced anyway.

The failing LV test is here, will update it when the fix is in develop

### CODE REVIEW

File: packages/manager/src/features/ObjectStorage/BucketLanding/BucketDrawer.tsx [hunk 1]
```diff
@@ -11,17 +12,24 @@ interface Props {
...
 
+  const { replace } = useHistory();
+
+  const closeDrawer = React.useCallback(() => {
+    closeBucketDrawer();
+    replace('/object-storage/buckets');
+  }, [closeBucketDrawer, replace]);
+
   return (
     <Drawer onClose={closeBucketDrawer} open={isOpen} title="Create a Bucket">
```
Comment (Jskobos) [inline packages/manager/src/features/ObjectStorage/BucketLanding/BucketDrawer.tsx:28]: `closeBucketDrawer` here should be `closeDrawer`

**Review by tiffwong — APPROVED** [review]
Looks good! Only question I have is if it should say "Launch LISH Console" so that it's more clear on what it's supposed to do?

File: packages/manager/src/features/linodes/LinodesLanding/LinodeActionMenu.tsx [hunk 2]
```diff
@@ -150,7 +150,7 @@ export class LinodeActionMenu extends React.Component<CombinedProps, State> {
     return (closeMenu: Function): Action[] => {
       const actions: Action[] = [
         {
-          title: 'Launch Console',
+          title: 'LISH Console',
```
Comment (Jskobos) [inline packages/manager/src/features/linodes/LinodesLanding/LinodeActionMenu.tsx:153]: just delete this file lol

**Review by Jskobos — APPROVED** [review]

File: packages/manager/src/features/linodes/LinodesLanding/LinodeActionMenu.tsx (hunk 2 above)
Comment (johnwcallahan) [reply packages/manager/src/features/linodes/LinodesLanding/LinodeActionMenu.tsx:153]: Haha, didn't even realize it.

**Pull request merged**